hs/t/n/ui256_xor_l.txt

```diff
@@ -0,0 +1,8 @@
+reachc: error[RE0129]: ^ expects arguments of either UInt, UInt256, Bool, Digest, or Bytes of the same length, but received: UInt256 and UInt
```

Comment:
This is kind of confusing because Digest and Bytes work with UInt right hand sides. Maybe this message should be more specific about the valid combinations

Reply:
Thanks for volunteering to make it work ;) I've made a Jira task